src/components/header/view.tsx

```diff
@@ -123,6 +121,9 @@ const Header = () => {
         <Spacer />
         <Spacer />
         <Spacer />
+        <Center mr={"20px"}>
+          <LocaleSwitcher />
```

Comment:
skip this if supportedLangs is empty or contains a single lang. No switch if nothing to switch.

I'd suggest to code the 'if' within the LocaleSwitcher